src/invidious/routes/api/v1/videos.cr

```diff
@@ -136,7 +136,11 @@ module Invidious::Routes::API::V1::Videos
           end
         end
       else
-        webvtt = YT_POOL.client &.get("#{url}&fmt=vtt").body
+        uri = URI.parse("#{url}")
```

Comment:
You don't need to use string extrapolation here!

```suggestion
        uri = URI.parse(url)
```

Reply:
fixed, the commits is squashed